### DISCUSSION TITLE
Charge card | Delayed debit card | Deferred payment card

### DISCUSSION THREAD

**Discussion by orducom1** [firefly-iii/firefly-iii#9274] [Firefly III]
Hello everyone,

I can't find answers to my questions and I'm coming to see if I can get some explanations.

Here it is, I'm using a "delayed debit card" whose operation is as follows.

The outstanding balance of the card goes from the 21st of month M to the 20th of month M+1 and the amount spent in this period is debited on the 30th of month M+1.
So what I spend on the 24th of month M will be debited from my account on the 30th of month M+1.

When I make purchases on October 23, the amount will be debited on November 30. It's very comfortable management ;)

I created a "Credit Card" account in which I record my expenses made with my card. But after...?
What should I do next so that Firefly3 reflects the actual operation of my card?

Thank you

**Comment by JC5** [comment]
What i do (I have the same system) is create a transfer from my asset account to my credit card on that day, to make the credit card balance be zero again.

**Reply by orducom1** [reply]
Hi JC5,

Glad to be in touch with you.

Okay, I understand the idea. Have you automated this process in any way? Have you used the different date fields available in Firefly3 to manage the deferred payment?

I hope you don't mind me slipping this in here... when do you think you'll look into allowing access to a shared account (a joint account) by two (or more) user accounts?

In any case, thank you very much for your work... are you developing all by yourself?

**Reply by JC5** [reply]
For my credit card, the deferred payment is a sum of all the previous expenses, so it's not easy to automate (and honestly, I don't bother). Since you seem to have the same system, I would create or import each CC transaction on the day it actually happened, and create the transfer on the 30th.

As for multi-user support, [this is in the works](https://docs.firefly-iii.org/how-to/firefly-iii/features/multi-user/).

And yes! Occasionally people open PRs but most of the work is done by me :)

**Reply by orducom1** [reply]
Hi James,

Okay, I'll look into creating an automation so that all expenses with a payment date on the 30th of the month are transferred from the "credit card" account to the current account... if that's possible ;) What do you think so ?

For the "multi-user" mode, I was thinking of access by two (or more) users to the same shared account (my wife and I on our joint account).

In any case, well done for the work accomplished... alone! That's impressive.

**Reply by JC5** [reply]
That could work yes. Myself, I made a transfer for the total sum instead, but I never bothered automating it.

Thanks for the compliments!

**Reply by Nutellaeis** [reply]
Sorry for reviving that old thread but this seems exactly what I also want to do. Did one of you figure out how to automate this? I am using the same system as James but this really seems like a candidate for automation. The rule would simply be a transaction with a value of "credit_card_balance" but currently it seems impossible to access acount balances in rules?
